README.md

```diff
@@ -180,16 +181,28 @@ A curated list of awesome resources related to the Ada and SPARK programming lan
 
 ## Compilers
 
-#### Local
-- [gnat-ce](https://www.adacore.com/download) - Adacore compiler for the Ada programming language which forms part of the GNU Compiler Collection. It supports all versions of the language, i.e. Ada 2012, Ada 2005, Ada 95 and Ada 83.
-- [fsf-gnat](http://www.getadanow.com/) - Free Software Foundation compiler for the Ada programming language which forms part of the GNU Compiler Collection. It supports all versions of the language, i.e. Ada 2012, Ada 2005, Ada 95 and Ada 83.
+#### Commercial / Closed source
```

Comment:
Commercial and closed source are two different things. I don't see why they should be grouped under the same category.

Also GNAT pro is missing from the list and it is a good example of commercial and open-source.

Reply:
Commercial generally means closed source and those compilers listed under that category are both.

As for GNAT Pro, then maybe there should be a Commercial / Open source category added? The fact that GNAT Pro wasn't there, is the reason that there is no category for it.

Reply:
@Fabien-Chouteau Can you edit to add the proper GNAT pro and decide which are commercial closed and commercial opened?

Reply:
Already done.

Reply:
@Fabien-Chouteau I will merge this one. If/when you have the time, please make a pr adding GNAT-Pro and a more detailed caterogy for Commercial, close and Commercial, open.
Thx!

Reply:
Maybe that is better, uses bold in parentheses.